ads/beaverads.js

```diff
@@ -23,13 +23,19 @@ import {loadScript, validateData} from '../3p/3p';
 export function beaverads(global, data) {
   validateData(data, ['blockId']);
 
-  const url = 'https://code.beaverads.com/data/' +
-      encodeURIComponent(data['blockId']) + '.js?async=1&div=c';
-
-  loadScript(global, url, () => {
-    global.context.renderStart();
-  }, () => {
-    global.context.noContentAvailable();
-  });
+  const url =
```

Comment:
Not sure why but a Prettier check was complaining about this

Reply:
See https://github.com/ampproject/amphtml/pull/22281#issuecomment-494153117

Reply:
Ah cool, thanks!